eulxml/xmlmap/core.py

```diff
@@ -65,14 +68,9 @@ def parseString(string, uri=None):
 _loaded_schemas = {}
 
 
-def loadSchema(uri, base_uri=None, override_proxy_requirement=False):
```

Comment:
The removal of all this http proxy stuff should probably be mentioned in the change log - and in particular, we should probably document that this parameter has been removed.